vagga.yaml

```diff
@@ -0,0 +1,99 @@
+containers:
+  mysql:
+    setup:
+      - !Alpine v3.4
+      - !Install
+        - mariadb
+        - mariadb-client
+      - !EnsureDir /data
+    environ: &db_config
+      DB_DATABASE: gdg
+      DB_USERNAME: mysql
+      DB_PASSWORD: mysql
+      DB_HOST: 127.0.0.1
+      DB_PORT: 3307
+      DB_DATA_DIR: /data
+    volumes:
+      /data: !Tmpfs
+        size: 200M
+        mode: 0o700
+
+  app:
+    setup:
+      - !Ubuntu xenial
+      - !UbuntuUniverse
+      - !Install
+        - python3.5
+        - python3.5-dev
+        - ca-certificates
+      - !BuildDeps
+        - git
+        - mercurial
+      - !PipConfig
+        dependencies: true
+        python-exe: python3.5
+      - !Depends setup.py
+      - !Depends requirements/common.txt  # As to https://github.com/tailhook/vagga/issues/236
+      - !Py3Requirements requirements/dev.txt
+      - !NpmInstall [bower]
+      - !Sh bower install --allow-root
+      - !EnsureDir /mnt/db_host
+    environ-file: /work/environ  # https://vagga.readthedocs.io/en/latest/container_params.html#opt-environ-file
+
+  test:
```

Comment:
@anxolerd learn about YAML anchors feature, it will help avoid lots of duplications: https://learnxinyminutes.com/docs/yaml/